app/forms/ConsigneeExciseFormProvider.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms
+
+import forms.mappings.Mappings
+import play.api.data.Form
+
+import javax.inject.Inject
+
+class ConsigneeExciseFormProvider @Inject() extends Mappings {
+
+  private val maxLength = 16
+
+  def apply(enableDynamicError: Boolean): Form[String] = {
```

Comment:
`enableDynamicError` doesn't really make sense to me. Would be better to name it like you did in the controller maybe. So `isNorthernIrishTemporaryRegisteredConsignee `